docs/pages/intro/faq.mdx

```diff
@@ -38,13 +38,13 @@ XMTP can be used with EVM-compatible wallet apps that support ECDSA signing on t
 
 The XMTP SDK **does not** include a wallet app abstraction, as XMTP assumes that developers have a way to obtain a wallet app connection.
 
-### Chains
+### Chains 
```

Comment:
How might we need to update this section? Because this section is so specific to EOAs and SCW, I think it is OK as-is. But we will need to update once we add other chains, like Bitcoin or Solana, yes?